solution_task/index.js

```diff
@@ -0,0 +1,8 @@
+const arr = [1, 2, 3, 4, 5];
+let mass;
+tail=(arr)=>{
+arr.shift()
+return arr
```

Comment:
I would do with slice or at least with creating a copy of array, to prevent mutating